packages/cozy-procedures/src/index.js

```diff
@@ -6,15 +6,15 @@ import Overview from './containers/Overview'
 import Amount from './components/Amount'
 import Duration from './components/Duration'
 import PersonalDataForm from './containers/PersonalDataForm'
-import Documents from './components/Documents'
+import DocumentsDataForm from './containers/DocumentsDataForm'
 
 const injectProcedureRoutes = ({ root }) => (
```

Comment:
Nit: `inject` is IMHO no the right word. When `inject`ing, I expect a "receiving" object being modified. Here, routes are merely rendered.